src/api/donations/entities/payment.entity.ts

```diff
@@ -0,0 +1,70 @@
+import { Field, Float, Int, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+class PaymentPayer {
+  @Field(() => String, { nullable: true })
```

Comment:
is it ok that first and last name are nullable?

Reply:
Yes, if the donor doesn't provide any.